src/containers/MissionControl.js

```diff
@@ -198,11 +263,33 @@ class MissionControl extends Component {
                     />
                   </Typography>
                 </ExpansionPanelSummary>
+                <Divider />
+                <ExpansionPanelDetails>
+                  <Typography>
+                    <FormattedMessage
+                      id="app.mission_control.buttons"
+                      description="Describes the buttons sensor section"
+                      defaultMessage="Buttons"
```

Comment:
Note: I added this label here to show what it will look like, but I did *not* yet change the functionality of these indicators. They are still hooked up to change with the light sensor values. We'll need to change them to show "button has been pressed" in the next PR.